### DISCUSSION TITLE
Volvió a no funcionar report en producto de mantenimiento

### DISCUSSION THREAD

**Discussion by nicolasdurans** [Cotalker/documentation#777] [4. Bug Reports] [answered]
![image](https://user-images.githubusercontent.com/5656833/158898479-109f53f5-d3f9-4ca5-a954-4c25841eb06d.jpeg)

**Comment by acedward** [comment, marked as answer]
https://github.com/Cotalker/documentation/issues/783

**Reply by acedward** [reply]
No pudo ser reproducido.
